File: 250827/중앙값 구하기/find-the-median.py
A, B, C = map(int, input().split())

# A. B인 경우 
if A < B:
    if B < C: 
        print(B) # A, B, C
    elif C < A:
        print(A) # C, A, B
    else: 
        print(C) # A, C, B
# B, A인 경우 
else: 
    if B > C: 
        print(B) # C, B, A
    elif C > A:
        print(A) # B, A, C
    else:
        print(C) # B, C, A

File: 250827/중앙값 구하기/find-the-median.js
const fs = require("fs");
let [A, B, C] = fs.readFileSync(0).toString().trim().split(" ").map(Number);

// A, B인 경우 
if (A < B) {
    if (B < C) {
        console.log(B); // A, B, C
    } else if (A < C) {
        console.log(C); // A, C, B
    } else {
        console.log(A); // C, A, B
    }
}
 // B, A인 경우         
 else {
    if (A < C) {
        console.log(A); // B, A, C
    } else if (B < C) {
        console.log(C); // B, C, A
    } else {
        console.log(B); // C, B, A
    }
}